conductor/lib/maya_utils.py

```diff
@@ -472,21 +472,31 @@ def get_render_layers_info():
     return render_layers
 
 
-def collect_dependencies(node_attrs):
+def collect_dependencies(node_attrs, leaf_path_list=None, exclude_path_list=None):
```

Comment:
Remove the `_list` suffix in the variable names, and pluralize. This is a bad practice in Python, and we now have type hints to help.

Reply:
Fixed